db/migrate/20220207132850_create_operations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOperations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :operations do |t|
+      t.string :name
+      t.float :amount
+      t.references :user, null: false, foreign_key: true
+      
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Kindly note that the reference column should be `t.references :author, foreign_key: { to_table: 'users' }`.  This will create `author_id` and a corresponding index.